instructure/canvas-ios
# Add people context card for groups

refs: MBL-15228
affects: Student
release note: Fixed user profile not opening from group menu.

test plan:
- Log into an account which has an account level group added
- Enter Group > People
- Tap on a user's row
- User profile page should open
